src/index.js

```diff
@@ -417,7 +426,7 @@ function swap (arr, i, n) {
 }
 
 
-function changeOrientation(contour) {
+function changeOrientation(contour) {  // eslint-disable-line no-unused-vars
```

Comment:
Why keep this method?
1. Not being used
2. Only applying `.reverse()` (nothing fancy here)